recipes/clmm/meta.yaml

```diff
@@ -0,0 +1,49 @@
+{% set name = "clmm" %}
+{% set version = "1.14.40" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.org/packages/source/{{ name[0] }}/{{ name }}/clmm-{{ version }}.tar.gz
+  sha256: edfab4226e7eba3a21b5cf938d5ac1a72cedfa020c6c6d9796e1021a99b13afc
+
+build:
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation
+  number: 0
+
+requirements:
+  host:
+    - python >=3.9,<=3.12
+    - hatchling
+    - pip
+  run:
+    - python >=3.9,<=3.12
+    - astropy >=4,!=5.0
+    - matplotlib-base
+    - numpy >=1.17
+    - scipy >=1.3
+    - healpy
+    - qp-prob
+    - pyccl
```

Comment:
If this is an optional backend, it may be best to put it under `test`/`requires`

Reply:
It is optional, but if there is no backend installed, the `import` test will fail